src/browser/navigation_validation.ts

```diff
@@ -13,20 +13,20 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-let coreState = require('./core_state.js');
-let electronApp = require('electron').app;
-let SubScriptionManager:any = require('./subscription_manager.js').SubscriptionManager;
-let subScriptionManager:any = new SubScriptionManager();
+const coreState = require('./core_state.js');
```

Comment:
Happy to, but as general rule, I am now trying to change the minimum necessary to address the ticket at hand. So let's leave that for a refactor ticket.